api/create_tables.sql

```diff
@@ -0,0 +1,56 @@
+CREATE DATABASE IF NOT EXISTS cs348db;
+USE cs348db;
+
+CREATE TABLE IF NOT EXISTS UnitListerAccount(
+    pm_id int NOT NULL AUTO_INCREMENT,  
+    name VARCHAR(255) NOT NULL,
+    phone_num VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL,
+    website VARCHAR(255) NOT NULL,
+    PRIMARY KEY (pm_id)
+    );
+
+CREATE TABLE IF NOT EXISTS Building(
+    building_id int NOT NULL AUTO_INCREMENT,
+    address VARCHAR(255) NOT NULL,
+    pet_friendly TINYINT(1) NOT NULL, -- TINYINT functions as a boolean
+    laundry_availability VARCHAR(255) NOT NULL, -- Do we want a yes or no, or ensuite vs in building
```

Comment:
If we decide to go with ensuite, building or none, we might want to make this an enum
```suggestion
    laundry_availability ENUM("building", "ensuite", "none") NOT NULL,
```